bears/python/PycodestyleBear.py

```diff
@@ -35,7 +35,7 @@ def create_arguments(
         :param max_line_length:
             Limit lines to this length.
         """
-        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s'"]
+        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s"]
```

Comment:
@jayvdb I didn't notice that... :worried: 

@sahilarora535 ```'``` is not even fixing the problem... have you actually tested it locally before submitting PR ?